build-system/specifications.md

```diff
@@ -0,0 +1,4606 @@
+without a `PROD_OBJLIBS_<osclass>` definition specified.
+
+```
+<prodname>_OBJLIBS += <name>
+```
+
+Combined object files will be used for all builds of the named
+product.
+
+```
+<prodname>_OBJLIBS_<osclass> += <name>
+```
+
+Combined object files will be used in builds of the named product for
+archs with the specified osclass.
+
+```
+<prodname>_OBJLIBS_DEFAULT += <name>
+```
+
+Combined object files will be used in builds of the named product for
+archs without a `<prodname>_OBJLIBS_<osclass>` definition
+specified.
+
+```
+<prodname>_LDOBJS += <name>
+```
+
+Object files will be used for all builds of the named product.
+(deprecated)
+
+```
+<prodname>_LDOBJS_<osclass> += <name>
+```
+
+Object files will be used in builds of the name product for archs with
+the specified osclass. (deprecated)
+
+```
+<prodname>_LDOBJS_DEFAULT += <name>
+```
+
+Object files will be used in builds of the product for archs without a
+`<prodname>_LDOBJS_<osclass>` definition specified. (deprecated)
+
+#### Specifying product source file names
+
+Source file names, which must have a suffix, are defined as follows:
+
+```
+SRCS += <name>
+```
+
+Source files will be used for all defined libraries and products.
+
+```
+SRCS_<osclass> += <name>
+```
+
+Source files will be used for all defined libraries and products for
+all archs of the specified osclass.
+
+```
+SRCS_DEFAULT += <name>
+```
+
+Source files will be used for all defined libraries and products for
+any arch that does not have a `SRCS_<osclass>` definition
+
+```
+USR_SRCS += <name>
+```
+
+Source files will be used for all products and libraries.
+
+```
+USR_SRCS_<osclass> += <name>
+```
+
+Source files will be used for all defined products and libraries for
+all archs of the specified osclass.
+
+```
+USR_SRCS_DEFAULT += <name>
+```
+
+Source files will be used for all defined products and libraries for
+any arch that does not have a `USR_SRCS_<osclass>` definition
+
+```
+PROD_SRCS += <name>
+```
+
+Source files will be used for all products.
+
+```
+PROD_SRCS_<osclass> += <name>
+```
+
+Source files will be used for all defined products for all archs of
+the specified osclass.
+
+```
+PROD_SRCS_DEFAULT += <name>
+```
+
+Source files will be used for all defined products for any arch that
+does not have a `PROD_SRCS_<osclass>` definition
+
+```
+<prodname>_SRCS += <name>
+```
+
+Source file will be used for the named product.
+
+```
+<prodname>_SRCS_<osclass> += <name>
+```
+
+Source files will be used for named product for all archs of the
+specified osclass.
+
+```
+<prodname>_SRCS_DEFAULT += <name>
+```
+
+Source files will be used for named product for any arch that does not
+have a `<prodname>_SRCS_<osclass>` definition
+
+#### Specifying libraries to be linked when creating the product
+
+For each library name specified which is not a system library nor a
+library from EPICS base, a <name>_DIR definition must be present in
```

Comment:
\<name\>_DIR -> \`\<name\>_DIR\ `
(right now \<name\> is not appearing on rendered version)